services.go

```diff
@@ -140,7 +140,7 @@ func (s *serviceSet) handle(ctx context.Context, req *Request, respond func(*sta
 			respond(st, p, stream.StreamingServer, true)
 		}()
 
-		if req.Payload != nil {
+		if req.Payload != nil || !info.StreamingClient {
```

Comment:
Can you add a comment here?
I'm not sure I follow why this fixes the issue.

Reply:
Added https://github.com/containerd/ttrpc/pull/157/commits/44ca0096e1b2b638ead24b4898cc34aa3b603926

Reply:
Thanks for the comment, I'm not sure how that relates to`!info.StreamingClient`
I guess this comes down to the `info` struct not really being documented.

So this is the server side and we are forcing it to deliver the empty payload?

Reply:
This covers one additional case of client streaming.

This one works:
```go
rpc DivideStream(Sum) returns (stream Part);
```

However if you change signature to:
```go
rpc DivideStream(google.protobuf.Empty) returns (stream Part);
```
It won't work, because `google.protobuf.Empty` translated into nil payload, so the `DivideStream` call skipped.